guides/common/modules/proc_running-installer.adoc

```diff
@@ -3,17 +3,62 @@
 [id="running-the-{project-context}-installer_{context}"]
 = Running the {Project} installer
 
-The Foreman installer is a collection of Puppet modules that installs {Project} based on native operating system packages.
-
-The installation run is non-interactive by default.
-The script displays its progress and writes logs to `{installer-log-file}`.
+Install the required packages and run the {Project} installer utility.
 
 .Procedure
-* Run the {Project} installer:
+ifdef::foremanctl[]
+[tabs]
+====
+Puppet-based installation::
+endif::[]
+. Upgrade all packages:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-upgrade}
+----
+. Install {project-installer-package}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install} {project-installer-package}
+----
+. Run the {Project} installer:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {installer-scenario}
 ----
 +
+The {foreman-installer} installer is a collection of Puppet modules that installs {Project} based on native operating system packages.
++
+The installation run is non-interactive by default.
+The script displays its progress and writes logs to `{installer-log-file}`.
++
 For a list of the available customization options, run `{foreman-installer} --help` or `{foreman-installer} --full-help`.
+ifdef::foremanctl[]
+Containerized installation::
+. Upgrade all packages:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-upgrade}
+----
+. Install the `foremanctl` package:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf install foremanctl
+----
+. Run the installer:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foremanctl deploy
+----
++
+You can find credentials to access your {ProjectServer} in the message shown in the `TASK [post_install : Admin credentials]` task results.
++
+For a list of available options, run `{foremanctl} --help`.
```

Comment:
Why is this an attribute but the other occurences above aren't rendered from the attribute?
Also, this attribute has an empty value from `master.adoc`.